ratis-server/src/main/java/org/apache/ratis/server/impl/RaftServerProxy.java

```diff
@@ -324,6 +338,7 @@ public void close() {
 
       try {
         getServerRpc().close();
+        getDataStreamServerRpc().close();
```

Comment:
It should have its own try-catch so that getDataStreamServerRpc() will be closed if getServerRpc().close() throws an exception.